README.md

```diff
@@ -79,36 +78,45 @@ adyen.checkout.version = 50
 - payment_session
 - payments.result
 - payment_methods
+- payment_methods.balance
 - payments
 - payments.details
 - payment_links
-
-**checkout utility:**
+- payment_links.get
+- payment_links.update
 - origin_keys
+- orders
+- orders.cancel
 
 **payments:**
 - authorise
 - authorise3d
+- authorise3ds2
+- get_authentication_result
+- retrieve3ds_result
```

Comment:
Let's add an underscore before 3ds2 to keep it consistent - so `retrieve_3ds2_result`.

Reply:
Ah.. that's a good point.  Similar to the other comment, we shouldn't make breaking changes without advanced notice, so we have to leave the authoriseXX endpoints as is.  Meaning that this endpoint has to be inconsistent with one or the other.

With that in mind, I'd say let's call it `retrieve_3ds2_result` and stay consistent moving forward.